### ISSUE TITLE
PinchGestureHandler not working on "recycled" cells

### ISSUE THREAD

**Issue by timurridjanovic** [Flipkart/recyclerlistview#518]
PinchGestureHandler doesnt quite work with recycling enabled. I want to be able to zoom into an image but with recycling enabled, it works for the first few images but then wont work with the "recycled cells".
Here is a recording of what I mean: https://recordit.co/aTrIETDTrb

I can dysableRecycling to make it work, but that kind of defeats the purpose of using recyclerlistview.. 
Any idea on how to get around that?

**Comment by akay25** [comment]
Hi @timurridjanovic ,
Did you find any good solution to this?

Cause I'm also working on some module in which I need to display 13-60 images full screen like a gallery, with zoom enabled and swipe as well.
But can't find a good approach.

**Comment by timurridjanovic** [comment]
@akay25 
Not really.. instead of using pinch gesture on the recyclerListView, I open a full screen modal with the image in it, and use the pinch gesture on that.. Basically, can't replicate the instagram pinch gesture model, but can replicate the facebook one (Facebook opens a modal)
